patches/sitecustomize.py

```diff
@@ -6,7 +6,7 @@
     from google.auth import credentials
     from google.cloud import bigquery
     from google.cloud.bigquery._http import Connection
-    from kaggle import PublicBigqueryClient
+    from kaggle_gcp import PublicBigqueryClient
```

Comment:
I suppose this should be `from kaggle.gcp import ...`, once that works (VMs are updated across the board), so to reflect the expected general usage?   => :Leave a TODO or make that change and hold until we're ready for it.

Reply:
We've discussed offline. The issue here is that the tests would fail if we did this since the worker is the one who imports kaggle_gcp into the kaggle.gcp namespace, and since that doesn't happen during the tests this would fail. I've added a comment todo to correct this.